odoo/tools/translate.py

```diff
@@ -1470,28 +1470,33 @@ def load_language(cr, lang):
     installer.lang_install()
 
 
+
+def get_po_paths(module_name: str, lang: str):
+    lang_base = lang.split('_')[0]
+    if lang_base == 'es' and lang != 'es_ES':
```

Comment:
We don't have es_ES yet, do we? Should we start using es_ES instead of es for Castilian Spanish?

Reply:
We do, the `code` of `base.lang_es` is `es_ES`.